forge/db/migrations/20231207-01-add-to-flow-template-table.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-unused-vars */
+
+const { DataTypes, QueryInterface } = require('sequelize')
+
+module.exports = {
+    /**
+     * Add icon and order to FlowTemplate (renamed FlowBlueprints in the UI)
+     * @param {QueryInterface} context Sequelize.QueryInterface
+     */
+    up: async (context) => {
```

Comment:
As FlowTemplate is an EE model, this needs to check the table exists before attempting to modify it.

Reply:
[787605b](https://github.com/FlowFuse/flowfuse/pull/3180/commits/787605b85a264e50b27fc591135ad1a998ec2370)